src/it/scala/org/broadinstitute/dsde/rawls/WorkspaceSimulation.scala

```diff
@@ -44,7 +53,7 @@ class WorkspaceSimulation extends FlatSpec with WorkspaceApiService with EntityA
   def actorRefFactory = system
 
   // convenience methods - TODO add these to unit tests too?
-  def addCookie = addHeader(HttpHeaders.`Cookie`(HttpCookie("iPlanetDirectoryPro", "test_token")))
+  def addMockOpenAmCookie = addHeader(Cookie(HttpCookie("iPlanetDirectoryPro", "test_token")))
```

Comment:
Hm.  The other one is a *real cookie* for a *test user* so I don't know if that distinguishes the two enough.